webpack/scenes/ModuleStreams/Details/Repositories/TableSchema.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  headerFormatter,
+  cellFormatter,
+} from '../../../../move_to_foreman/components/common/table';
+import helpers from '../../../../move_to_foreman/common/helpers';
+
+const TableSchema = [
+  {
+    property: 'name',
+    header: {
+      label: __('Name'),
+      formatters: [headerFormatter],
+    },
+    cell: {
+      formatters: [
+        (value, { rowData }) => (
+          <td>
+            <Link to={helpers.urlBuilder(`products/${rowData.product_id}/repositories`, '', rowData.id)}>{rowData.name}</Link>
```

Comment:
You should use a standard `<a href>` here instead of the `<Link>` component.  I believe the `<Link>` component is only for `react-router` urls.